bundle/src/main/java/com/adobe/acs/commons/models/injectors/annotation/ParentResourceValueMapValue.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * #%L
+ * ACS AEM Commons Bundle
+ * %%
+ * Copyright (C) 2020 Adobe
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.adobe.acs.commons.models.injectors.annotation;
+
+import org.apache.sling.models.annotations.Source;
+import org.apache.sling.models.annotations.injectorspecific.InjectionStrategy;
+import org.apache.sling.models.spi.injectorspecific.InjectAnnotation;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/**
+ * Annotation to be used on either methods, fields to let Sling Models inject an inherited property from the
+ * parent resource
+ */
+@Target({METHOD, FIELD, PARAMETER})
+@Retention(RUNTIME)
+@InjectAnnotation
+@Source(ParentResourceValueMapValue.SOURCE)
+public @interface ParentResourceValueMapValue {
+
+    String SOURCE = "parent-valuemap-value";
+
```

Comment:
We should add the ability to name the field, in case the variable name doesnt match the property name.  This is standard on other similar annotations.

`
    String name() default StringUtils.EMPTY;
`